### ISSUE TITLE
Windows dialog box format

### ISSUE THREAD

**Issue by juniperlsimonis** [jeroen/askpass#10]
First of all, thank y'all for a really helpful package!

I'm currently working with folks in a Windows-only environment and am using the Credentials Manager to store creds for data storage after first entry to minimize re-typing, and relying on the keyring package that sources askpass for the actual secret interactions. So that's what brings me here. 

I have a specific question regarding the dialog box for the Windows cred input, which as is seen on the [README page](https://github.com/r-lib/askpass#rgui--windows), has more content than the other UI/OS dialogs, which have a single prompt and a single field. The windows one, however, has a box title, prompt, user name field filled in with NA, and then a forced 'password' field.

I've looked at the c files and can see where the [forced username na](
https://github.com/r-lib/askpass/blob/6d2f96d1016b386620edfff8387c539b3d404890/src/win32/win-askpass.c#L22) and [forced box title](https://github.com/r-lib/askpass/blob/6d2f96d1016b386620edfff8387c539b3d404890/src/win32/win-askpass.c#L31) happen, but can't see where the forced "password:" prompt comes from.

So I was wondering (recognizing that the answers might not be the same for all three of those spots)

1. Is there a reason for the formatting of the windows dialog box being the way it is, especially in relation to the others?
2. If there's no reason except for that's it's current state, are you up for requests / suggestions / PRs with edits in those files? 

I ask because I'd really like to streamline this dialog box to minimize the need for explainers ("ignore the user name field", "yes it says password but we're actually asking for your login id to store it") for end users.

Thanks much again!

**Comment by jeroen** [comment]
Yes I am open for improvements. At the time, this was the best thing I could get to work on different versions of Windows. I could not find a native way in Windows to just display a password entry box, without a username field.
